components/org.wso2.analytics.is/src/main/java/org/wso2/analytics/is/siddhi/extension/IpToCityExtension.java

```diff
@@ -45,7 +44,7 @@ protected Object execute(Object[] objects) {
     @Override
     protected Object execute(Object o) {
         try {
-            return geoLocationResolverUDF.getCity(o.toString());
+            return GeoResolverInitializer.getInstance().getLocationResolver().getLocation((o.toString()));
```

Comment:
We can move "GeoResolverInitializer.getInstance().getLocationResolver()" to init() or start() method. Please create a git issue.

Reply:
Created https://github.com/wso2/analytics-is/issues/349